### PR TITLE
Add macro & ODS assignment

### PR DESCRIPTION
**Pull request by Ingridds**

### CODE REVIEW

File: assignment/ingrid.sas [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    %mend means;
+
+%means(&ds, meanSD);
+%means(&ds, meanSD, class=status);
+
+*univariate*;
+%macro univariate (data, variable);
+proc univariate data= &data normal plots;
+var &variable;
+run;
```
Comment (lwjohnst86) [inline assignment/ingrid.sas]: I would recommend using ODS output for this, as it splits out a lot of stuff...

File: assignment/ingrid.sas [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+	ods listing close;
+	proc logistic data=&data descending;
+        class &class;
+	model &y = &x &class
+			/clodds=wald ;
+		where &where;
+	ods output NObs=obs
+		ResponseProfile=response
+		CLoddsWald=ORCI;
+    run;
```
Comment (lwjohnst86) [inline assignment/ingrid.sas]: Excellent! :)

**Comment by lwjohnst86** [comment]
Excellent job Ingrid :smile: 

A few things I would recommend: Have a consistent tab/spacing for your code. It was difficult for me to quickly read your code as some commands were indented properly but then the next would not be indented (eg: line 38 or 63). I would also generally not recommend using the actual tab character, but rather use spaces (4 or 2 spaces for a tab; tabs in computers converted into the `\t` character, which differs across operating systems in its length, while a space is always a space). Always tab/spaces after a `proc` or `data` statement, keeping the tab/space the same until the `run;` command. Also tab/space if the statement hasn't finished (`;`) on one line (eg: 63 to 64).

Last thing before I merge your assignment, can you remove the `.DS_Store` file?

Thanks again Ingrid!

**Comment by Ingridds** [comment]
Thanks for the input! I didn't know about the tab character. Good to know! And i'll take out the DS_Store

**Comment by lwjohnst86** [comment]
Hmm, nope, the `.DS_Store` file still exists. And now there is another one in the Thoughts/ folder!

Run this command:

```
git config --global core.excludesfile "C:\Users\yournameoraccountname\.gitignore"
cd ~
echo '.DS_Store' >> ./.gitignore
```

That should remove them from being tracked by Git. Then do:

```
cd ~/path/to/sandbox
git rm .DS_Store Thoughts/.DS_Store
## Make sure they are removed
git status
git commit -m "Removed .DS files"
git push
```

**Pull request merged**